PSKoans/Private/Get-TempDirectory.ps1

```diff
@@ -0,0 +1,36 @@
+﻿function Get-TempDirectory {
+    <#
+    .SYNOPSIS
+        Finds the temp directory on the current OS.
+
+    .DESCRIPTION
+        Finds the temp directory on the current OS, appends the filename PSKoans.tmp and returns a string of the path.
+
+    .EXAMPLE
+        Get-TempDirectory
+
+        Returns C:\Users\PSKoans\AppData\Local\Temp\PSKoans.tmp
+    #>
```

Comment:
Looks good, apart from the mentioned complication with private functions...

If this is going to be a public function, we'll need to generate markdown help for this with PlatyPS so that it creates the proper external help MAML for this function as well. You could do this or I can take care of it as a follow up. 🙂 

Assuming we stick with this course, at least. `TestDrive:` might be a simpler way to go, depending...

Reply:
Agreed, TestDrive seems the simpler approach. In rereading #174 the issue is less about using TestDrive and more about not knowing the contents of the file. The addition of the here string *should* resolve that confusion. 

So I would vote we get rid of get-tempdirectory and leverage testdrive.

Reply:
Sounds good to me. 😄